Util/ai_command.py

```diff
@@ -39,7 +40,7 @@ def addMoveTo(self, target: [Pose, Position], cruise_speed=1, end_speed=0, ball_
         self._pathfinder_on = True
         return self
 
-    def addKick(self, kick_force=1):
+    def addKick(self, kick_force:KickForce=KickForce.LOW):
         self._kick_force = kick_force
```

Comment:
Assert if we pass a KickForce?

Reply:
Yup, would be useful for the people who are unaware of the change